interface/locales/de/common.json

```
@@ -503,21 +503,21 @@
   "rescan_directory": "Verzeichnis neu scannen",
   "rescan_location": "Standort neu scannen",
   "reset": "Zurücksetzen",
-  "reset_and_quit": "Reset & Quit App",
-  "reset_confirmation": "Are you sure you want to reset Spacedrive? Your database will be deleted.",
-  "reset_to_continue": "We detected you may have created your library with an older version of Spacedrive. Please reset it to continue using the app!",
-  "reset_warning": "YOU WILL LOSE ANY EXISTING SPACEDRIVE DATA!",
+  "reset_and_quit": "Zurücksetzen und Verlassen",
+  "reset_confirmation": "Bist du Sicher das du Spacedrive Zurücksetzen willst? Deine datenbank wird Gelöscht.",
+  "reset_to_continue": "Es wurde Festgestellt das du deine Bibliothek mit einer älteren Version von Spacedrive erstellt hast. Bitte Zurücksetzen um die App weiterhin zu Verwenden.!",
+  "reset_warning": "DU VERLIERST ALLE BESTEHENDEN SPACEDRIVE DATEN!",
```

```suggestion
  "reset_and_quit": "Zurücksetzen & App Verlassen",
  "reset_confirmation": "Bist du sicher, dass du Spacedrive zurücksetzen willst? Deine Datenbank wird gelöscht.",
  "reset_to_continue": "Es wurde festgestellt, dass du deine Bibliothek mit einer älteren Version von Spacedrive erstellt hast. Bitte zurücksetzen, um die App weiterhin zu verwenden.!",
  "reset_warning": "DU VERLIERST ALLE BESTEHENDEN SPACEDRIVE-DATEN!",
```
Verbs aren't capitalised, but there are some words that are indeed capitalised in the English version